packages/fast-components-react-base/src/slider/slider.spec.tsx

```diff
@@ -767,6 +768,21 @@ describe("Slider", (): void => {
         document.body.removeChild(container);
     });
 
+    test("direction is set after component has mounted", (): void => {
+        const container: HTMLDivElement = document.createElement("div");
+        document.body.appendChild(container);
+
+        const rendered: any = mount(
+            <Slider managedClasses={managedClasses} initialValue={50} />,
+            {
+                attachTo: container,
+            }
+        );
+
+        expect(rendered.state("direction")).toBe(Direction.ltr);
+        document.body.removeChild(container);
+    });
```

Comment:
@scomea should we add a test to ensure that if the direction is null the slider does not render?

Reply:
figured one out